src/commonTest/kotlin/exchange.dydx.abacus/app/manager/V4RestrictionsTests.kt

```diff
@@ -140,4 +142,77 @@ class V4RestrictionsTests {
             )
         }
     }
+
+    @Test
+    fun testGeoEndpointHandling() {
+        reset()
+
+        testChain!!.signCompliancePayload = """
+        {
+            "signedMessage": "1",
+            "publicKey": "1",
+            "timestamp": "2024-05-14T20:40:00.415Z"
+        }
+        """.trimIndent()
+
+        testRest?.setResponse(
+            "https://indexer.v4staging.dydx.exchange/v4/compliance/geoblock",
+            """
+                {
+                    "status": "CLOSE_ONLY",
+                    "reason": null,
+                    "updatedAt": "2024-05-14T20:40:00.415Z"
+                }
+            """.trimIndent(),
+        )
```

Comment:
@johnqh does it look right that I have to set this spoofed response  twice (see below)

I have to set it twice because the test factory removes a spoofed response after the first accessing:

```
override fun get(
        url: String,
        headers: IMap<String, String>?,
        callback: RestCallback,
    ) {
        if (url.contains("env.json")) {
            callback(mock.environments.environments, 200, null)
            return
        }
        if (!url.contains("localization")) {
            requests.add(url)
        }
        val response = responses[url]
        if (response != null) {
            responses.remove(url) // <---- here
            val code = parser.asInt(response)
            if (code != null) {
                callback(null, code, null)
            } else {
                callback(response, 200, null)
            }
        } else {
            callback(null, 404, null)
        }
    }
```